src/test/kotlin/org/rust/cargo/runconfig/producers/TestRunConfigurationProducerTest.kt

```diff
@@ -11,11 +11,13 @@ import org.rust.cargo.runconfig.test.CargoTestRunConfigurationProducer
 import org.rust.lang.core.psi.RsFile
 import org.rust.lang.core.psi.RsFunction
 import org.rust.lang.core.psi.ext.RsMod
+import org.rust.lang.core.psi.ext.cargoPackage
+import org.rust.openapiext.toPsiDirectory
 
 class TestRunConfigurationProducerTest : RunConfigurationProducerTestBase() {
     fun `test test producer works for annotated functions`() {
         testProject {
-            lib("foo", "src/lib.rs", """
+            lib("foo", "tests/lib.rs", """
```

Comment:
Do we really need this change (and similar in other tests)?

Reply:
Not really. Fixed.